cmd/manager/main.go

```diff
@@ -24,6 +24,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+	
+	pipelinev1alpha1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
```

Comment:
Does this package need to be copied to the vendor directory?

Reply:
It looks like the Travis build failed, I think that this does need to be vendored.

Reply:
added new vendor files
https://github.com/kabanero-io/kabanero-operator/pull/186/commits/250ac65664e5791515c914d4ca5ab812591a0702